### DISCUSSION TITLE
Where is Blueish Color BG Coming From

### DISCUSSION THREAD

**Discussion by jrock2004** [LazyVim/LazyVim#2585] [Q&A]
So no matter what theme I set it too, I see that blue-ish color bg. How can I change that?

![image](https://github.com/LazyVim/LazyVim/assets/655716/a0216be7-9a19-4cc1-b628-2d4d35d647e1)

**Comment by jrock2004** [comment]
Figured it out

**Reply by abiriadev** [reply]
What was the reason? Would you tell me? I am having the exact same issue.
